src/twisted/newsfragments/10105.doc

```diff
@@ -0,0 +1,2 @@
+ Sync API docs templates with pydoctor 21.2.2 release.
+ 
```

Comment:
I agree that your wording suggestion is slightly better, but I really don't want to go through another CI cycle of over half an hour to change one line that may or may not end up in the release notes one day...

Reply:
100% understand the decision and the pain :(
 I can update the release notes as a drive by on one of my PRs.

-------

Many thanks for merging this PR.

I have removed this branch... it should keep the repo a bit cleanup and auto-completion less noisy :)

-------

Regarding re-releasing the documentation. Please check with Craig.

If this is cherry-picked in the release branch I can update the documentation at https://twistedmatrix.com/documents/current/api/

For Read The Docs... there is not much we can do other than creating a  new 21.2.0.post1 tag to trigger a new build on the updated code... but since RTD is still WIP, I think that we can let it be :)

I have created this ticket to track the RTD migration progress https://twistedmatrix.com/trac/ticket/10117